src/d/actor/d_a_obj_ebomzo.cpp

```diff
@@ -4,118 +4,299 @@
 //
 
 #include "d/actor/d_a_obj_ebomzo.h"
-#include "d/d_procname.h"
+#include "dolphin/types.h"
+#include "d/d_com_inf_game.h"
+#include "f_op/f_op_actor_mng.h"
+
+typedef enum {
+    /*0*/ Ebomzo_Mode_Check = 0,
+    /*1*/ Ebomzo_Mode_Demo,
+    /*2*/ Ebomzo_Mode_Fall,
+} EbomzoMode;
+
+Mtx daObjEbomzo::Act_c::M_tmp_mtx; //.bss 80810060
+const char daObjEbomzo::Act_c::M_arcname[] = "Ebomzo";
+
+dCcD_SrcSph sph_check_src = {
+    /*0x00 mObjInf*/ {
+        /* > 0x00 mObj*/ {
+            /* >> 0x00 mFlags*/ 0x00000000,
+            /* >> 0x04 mSrcObjHitInf*/ {
+        /* >>> 0x04 mObjAt*/ {
+                    /* >>>> 0x04 mType*/ 0,
+                    /* >>>> 0x08 mAtp*/ 0,
+                    /* >>>> 0x0C mBase*/ {
+        /* >>>>> 0x0C mSPrm*/ 0
+    }
+},
+/* >>> 0x10 mObjTg*/{
+    /* >>>> 0x10 mType*/ 0x20,
+    /* >>>> 0x14 mBase*/ {
+        /* >>>>> 0x14 mSPrm*/ 9
+    }
+},
+/* >>> 0x18 mObjCo*/{
+    /* >>>> 0x18 mBase*/ {
+        /* >>>>> 0x18 mSPrm*/ 0
+     }
+}
+}
+},
+
+/* > 0x1C mGObjAt*/ {
+    /* >> 0x1C mSe*/ 0,
+    /* >> 0x1D mHitMark*/ 0,
+    /* >> 0x1E mSpl*/ 0,
+    /* >> 0x1F mMtrl*/ 0,
+    /* >> 0x20 mBase*/ {
+        /* >>> 0x20 mSPrm*/ 0
+    }
+},
+
+/* > 0x24 mGObjTg*/ {
+    /* >> 0x24 mSe*/ 0,
+    /* >> 0x25 mHitMark*/ 0,
+    /* >> 0x26 mSpl*/ 0,
+    /* >> 0x27 mMtrl*/ 0,
+    /* >> 0x28 mBase*/ {
+        /* >>> 0x28 mSPrm*/ 0
+    }
+},
+
+/* > 0x2C mGObjCo*/ {
+    /* >> 0x2C mBase*/ {
+        /* >>> 0x2C mSPrm*/ 0
+     }
+},
+
+},
+
+/*0x30 mSphAttr*/{
+    /* > 0x30 mSph*/{
+        /* >> 0x30 mCenter*/ {0,0,0},
+        /* >> 0x3C mRadius*/ 10.0f
+    }
+}
+};
 
 /* 00000078-0000012C       .text CreateHeap__Q211daObjEbomzo5Act_cFv */
-void daObjEbomzo::Act_c::CreateHeap() {
-    /* Nonmatching */
+int daObjEbomzo::Act_c::CreateHeap() {
+    J3DModelData* model_data = (J3DModelData*)dComIfG_getObjectRes(M_arcname, 4);
```

Comment:
the res folder has enums for every arc's list of files, so if you include d/res/res_ebomzo.h you can avoid the magic number like this:
```suggestion
    J3DModelData* model_data = (J3DModelData*)dComIfG_getObjectRes(M_arcname, EBOMZO_BDL_EBOMZO);
```